cmake/find_metasmt.cmake

```diff
@@ -0,0 +1,79 @@
+find_package(metaSMT REQUIRED CONFIG)
+message(STATUS "metaSMT_DIR: ${metaSMT_DIR}")
+list(APPEND KLEE_COMPONENT_EXTRA_INCLUDE_DIRS
+  "${metaSMT_INCLUDE_DIR}" ${metaSMT_INCLUDE_DIRS})
+# FIXME: We should test linking
+list(APPEND KLEE_SOLVER_LIBRARIES ${metaSMT_LIBRARIES})
+
+# THIS IS HORRIBLE. The ${metaSMT_CXXFLAGS} variable
```

Comment:
@ccadar Yes the metaSMT version used in our build scripts is not fixed. @hoangmle if you fix the build scripts to use a particular revision of meta-SMT then I'd happily make the change. Having a fixed version is important because it means we can have reproducible builds in TravisCI and in our Docker image.